### PR TITLE
mandatory cli api key configuration

### PR DESCRIPTION
**Pull request by sethkennedy**

### CODE REVIEW

File: capella_console_client/session.py [hunk 1]
```diff
@@ -159,8 +159,8 @@ def _cache_user_info(self):
     def _set_api_key_auth_header(self, api_key: Optional[str]):
         assert isinstance(api_key, str)
         api_key = api_key.strip()
-        if not api_key.startswith("Apikey"):
-            api_key = f"Apikey {api_key}"
+        if not api_key.lower().startswith("apikey"):
+            api_key = f"ApiKey {api_key}"
```
Comment (grmpflh27) [inline capella_console_client/session.py:163]: Is the capital **K** in  Api**K**ey intentional?

File: capella_console_client/cli/settings.py [hunk 2]
```diff
@@ -83,20 +83,23 @@ def limit():
 
 
 @app.command()
-def user():
```
Comment (grmpflh27) [inline capella_console_client/cli/settings.py:86]: thanks for dropping this

File: capella_console_client/session.py (hunk 1 above)
Comment (grmpflh27) [reply capella_console_client/session.py:163]: (I realize that we are case insensitive in our authorizer but think we should define a standard)

**Review by grmpflh27 — COMMENTED** [review]
looks good - thanks for putting this in

**Comment by grmpflh27** [comment]
@sethkennedy - can you please also update the CLI docs referenced here: https://capella-console-client.readthedocs.io/en/main/pages/wizard.html#configure (make livedocs is your friend here) - thanks

File: capella_console_client/session.py (hunk 1 above)
Comment (sethkennedy) [reply capella_console_client/session.py:163]: > Is the capital **K** in Api**K**ey intentional?

Yes. This is the standard this is being used in the target API code (even though it is not case sensitive).

**Review by grmpflh27 — APPROVED** [review]

**Pull request merged**